web/pdf_page_view.js

```diff
@@ -724,6 +730,7 @@ class PDFPageView {
       return Promise.reject(new Error("pdfPage is not loaded"));
     }
 
+    this.toggleLoadingIconSpinner(/* viewVisible = */ true);
```

Comment:
This doesn't seem correct, since it'd essentially re-introduce issue #14242 here; please remove this.
Note that we're already showing it in the standalone case, see above, and for the full viewer the actually visible pages should be determining this.

Reply:
As mentioned, please remove this since it shouldn't be necessary.

Reply:
I forgot to save the file... it's really time to take vacations.